lib/editor-features.php

```diff
@@ -0,0 +1,49 @@
+<?php
+/**
+ * Experimental editor features config functionality.
+ *
+ * @package gutenberg
+ */
+
+/**
+ * Returns the default config for editor features,
+ * or an empty array if none found.
+ *
+ * @return array Default features config for the editor.
+ */
+function gutenberg_experimental_get_editor_features_config() {
+	$empty_config = array();
+	$config_path  = __DIR__ . '/experimental-default-theme.json';
+	if ( ! file_exists( $config_path ) ) {
+		return $empty_config;
+	}
+
+	$theme_config = json_decode(
+		@file_get_contents( $config_path ),
```

Comment:
Is the `@` trick preferred to check if the file exists?

Reply:
Related discussion:

https://github.com/WordPress/gutenberg/pull/22291#discussion_r427982610

The check for file existence is earlier, it prevents other potential issues with the file listed by @aduth  :)